src/core/components/hv-root/index.tsx

```diff
@@ -79,15 +79,15 @@ export default class Hyperview extends PureComponent<HvScreenProps.Props> {
    */
   reload = (
     optHref: DOMString | null | undefined,
-    opts: HvComponentOptions,
     onUpdateCallbacks: OnUpdateCallbacks,
+    opts: HvComponentOptions | undefined,
   ) => {
     const isBlankHref =
       optHref === null ||
       optHref === undefined ||
       optHref === '#' ||
       optHref === '';
-    const stateUrl = onUpdateCallbacks.getState().url;
+    const stateUrl = onUpdateCallbacks?.getState().url;
```

Comment:
Is this is needed? `onUpdateCallbacks` was not changed to an optional (same question on other lines changed pertaining to `onUpdateCallbacks`)

Reply:
Reverted.